.github/workflows/ci-build.yml

```
@@ -0,0 +1,22 @@
+    if: ${{ github.actor != 'dependabot[bot]' }}
+    name: Build and analyze
+    runs-on: ubuntu-latest
+    permissions:
+      checks: write
+      pull-requests: write
+    steps:
+      - uses: bmazzarol/bmazzarol/.github/actions/ci-build@main
+        with:
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
+          sonarToken: ${{ secrets.SONAR_TOKEN }}
+          sonarProjectKey: bmazzarol_Polyglot.Notebook.Docfx.Plugin
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)